src/test/components/elements/ReportButton.test.tsx

```diff
@@ -0,0 +1,52 @@
+import { screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import { ReportButton } from "../../../app/components/elements/ReportButton";
+import { renderTestEnvironment } from "../../utils";
+import * as actions from "../../../app/state/actions/logging";
+
+const renderReportButton = (pageId?: string) => {
+  renderTestEnvironment({
+    role: "STUDENT",
+    PageComponent: ReportButton,
+    componentProps: {
+      pageId: pageId,
+    },
+    initialRouteEntries: ["/"],
+  });
+};
+
+const button = () =>
+  screen.getByRole("button", {
+    name: /report a problem/i,
+  });
+
+describe("ReportButton", () => {
+  jest.spyOn(window, "open").mockImplementation(jest.fn());
+
+  const logActionSpy = jest.spyOn(actions, "logAction");
+
+  const testCases = [undefined, "example_id"];
```

Comment:
Similarly to my comment in the other test file, it's better to name this after whatever you're trying to test.